### PR TITLE
Make min_error a minimum value and update README

### PR DESCRIPTION
**Pull request by joe-pitt**
Changed inferpymc so that it interprets min_error as a minimum value (lower bound) rather than adding it to the other errors for all points. This addresses Issue #116 .

Needs looking at by someone who understands pytensors before approval - I'm not certain that what I've done here is correct.

### CODE REVIEW

**Review by hdelongueville — APPROVED** [review]
I'm not an expert on pytensors but this change seems right to me.

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 1]
```diff
@@ -285,7 +285,7 @@ def inferpymc(
             pollution_event = np.abs(pt.dot(hx, x))
 
         pollution_event_scaled_error = pollution_event * sig[sites, sigma_freq_index]
-        epsilon = pt.sqrt(error**2 + pollution_event_scaled_error**2 + min_error**2)
+        epsilon = pt.maximum(pt.sqrt(error**2 + pollution_event_scaled_error**2), min_error)
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:288]:
```suggestion
        epsilon = pt.max(pt.sqrt(error**2 + pollution_event_scaled_error**2), min_error)
```

**Review by brendan-m-murphy — CHANGES_REQUESTED** [review]
I think the pytensor function is called `max`: https://pytensor.readthedocs.io/en/latest/library/tensor/basic.html#reductions although it's possible that `maximum` works too... their documentation isn't very good.

Otherwise, looks good!

File: openghg_inversions/hbmcmc/inversion_pymc.py (hunk 1 above)
Comment (joe-pitt) [reply openghg_inversions/hbmcmc/inversion_pymc.py:288]: Hmmm I think pt.maximum and pt.max do different things (https://pytensor.readthedocs.io/en/latest/library/tensor/basic.html#pytensor.tensor.maximum). I'm not certain that pt.maximum is right though, so you better check!

**Comment by brendan-m-murphy** [comment]
It looks like `pt.maximum` is what we want: https://pytensor.readthedocs.io/en/latest/library/tensor/basic.html#mathematical We could also achieve this with the `switch` statement.

**Review by brendan-m-murphy — APPROVED** [review]

**Pull request merged**